Automattic/jetpack
# My Jetpack: Use `ThemeProvider` in favor of `base-styles`.

This one is a follow-up PR about moving out from `base-styles` to `ThemeProvider` and component-driven approach: https://www.componentdriven.org/

#### Changes proposed in this Pull Request:

* Use `ThemeProvider` at `My Jetpack`.
* Remove `base-styles` usage.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Navigate to `My Jetpack` page.
* You should see everything normal, no visual changes.
